CreateGroup/function.json

```
@@ -0,0 +1,19 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
```

We use anonymous auth level in our project
```suggestion
      "authLevel": "anonymous",
```